testdata/p4_16_samples_outputs/issue3287-first.p4

```diff
@@ -0,0 +1,4 @@
+bit<4> func(in bit<4> l) {
```

Comment:
This function is optimized away, so it is difficult to tell how it is actually transformed.

Reply:
Can we make it so this test is not optimized away?